android/BOINC/app/build.gradle

```diff
@@ -122,18 +122,20 @@ android {
 }
 ext {
     coroutines_version = '1.3.8'
-    dagger_version = '2.28.1'
+    dagger_version = '2.29.1'
     lifecycle_version = '2.2.0'
-    powermock_version = '2.0.5'
-    junit5_version = '5.6.2'
+    powermock_version = '2.0.7'
+    junit5_version = '5.7.0'
 }
 
 dependencies {
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```

Comment:
Why'd you re-add this library?

Reply:
Sometimes things happen automatically.

Reply:
I checked and do see a double, think it's a Kotlin legacy thing.
I removed it and will commit it